app/scripts/views/mixins/avatar-mixin.js

```diff
@@ -83,8 +82,7 @@ define([
       }
     },
 
-    updateProfileImage: function (profileImage) {
-      var account = this.getSignedInAccount();
+    updateProfileImage: function (profileImage, account) {
```

Comment:
In `updateCachedProfileImage`, the signature is reversed. Would be nice for consistency between the two.